packages/nitro-protocol/package.json

```diff
@@ -10,6 +10,7 @@
     "ethers": "4.0.44"
   },
   "devDependencies": {
+    "@statechannels/client-api-schema": "0.0.1",
```

Comment:
It doesn't seem right to me that `nitro-protocol` has our client-api-schema as a dependency (and why as a devDependency anyway?)

Reply:
It is in `outcome.ts` for the `AllocationItem` definition. Maybe it shouldn't need this? `devDependency` because it is just a type and so isn't needed in a production build.

Reply:
Good point on the `devDependency` thing 👍 .

Reply:
Doesn't tree shaking take care of that for us?